tox.ini

```diff
@@ -1,11 +1,9 @@
 [tox]
 envlist =
     py27,
-    py27-datetime,
     py35,
     py36,
-    py36-datetime,
```

Comment:
This means that running `tox` locally no longer runs the `datetime` tests thus dropping the coverage. I think we should keep them. I did not try but I could imagine that it even works together with your changes to the `datetime` testenv.